### PR TITLE
Adds profiles to get started

### LINKED ISSUE #21: Add Recommended Git Config
We should have a recommended git config and instructions for copying it locally to the correct location.

### PR DESCRIPTION
**Pull request by jbteves**
Closes #21 
Bonus: adds a .vimrc

Curious to see if @handwerkerd @pmolfese or @agt24 have suggestions for configurations for users other than:
- git config
- .vimrc

I don't believe in telling people how to set up their shell rc-files.

### CODE REVIEW

**Comment by jbteves** [comment]
@handwerkerd or @pmolfese do you have a sample .bashrc you could supply?

**Comment by pmolfese** [comment]
# .bash_profile

# Get the aliases and functions
if [ -f ~/.bashrc ]; then
        . ~/.bashrc
fi

# User specific environment and startup programs

export PATH=$PATH:$HOME/bin #this is where i put random programs that i'm building like laynii

#shortcut for freesurfer after doing a module load freesurfer
alias freesurfer='source $FREESURFER_HOME/SetUpFreeSurfer.sh'

#mricron
export PATH=$PATH:/home/molfesepj/Applications/mricron_lx

#TORTOISE
#export PATH=$PATH:/home/molfesepj/Applications/TORTOISE_V3.1.0/DIFFPREPV31/bin/bin

#MNE
export MNE_ROOT=/home/molfesepj/Applications/MNE-2.7.3
alias mne='. /home/molfesepj/Applications/MNE-2.7.3/bin/mne_setup_sh'

#anaconda
alias conda3='source /data/molfesepj/conda3/etc/profile.d/conda.sh && conda activate base'

**Review by handwerkerd — COMMENTED** [review]
I think profiles/README.md needs a bit more context for people not familiar with this stuff. That is a couple of sentences like, "Profiles are files, often beginning with `.` that contain information a program uses every time it runs. A git profile is very useful because it sets up some consistent options. The .vimrc profile is useful if you plan to use the vim text editor. this directory also contains some scripts (ending with `.sh` that can place this profiles in the appropriate locations with some individualized settings.

**Comment by jbteves** [comment]
Okay, I'll add that. Thanks for the feedback.

File: profiles/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Profiles
+Profiles are files, often beginning with a `.` in Unix in order to hide
+them, which contain information for your terminal and terminal-based
+programs to access every time they run.
```
Comment (handwerkerd) [inline profiles/README.md:4]:
```suggestion
Profiles are files, often beginning with a `.` in which contain
information for your terminal and terminal-based programs to
access every time they run. This files are normally hidden when
you type `ls`. `ls -a` will show files beginning with a `.`
```

**Review by handwerkerd — APPROVED** [review]

**Pull request merged**